qualang_tools/utils/README.md

```diff
@@ -0,0 +1,53 @@
+# QUA Utils

```

Comment:
Need to think on the name :)
I actually think it's fine, but do we want to have everything here? what decides what goes here and what else where?

Reply:
Yes we really need to discuss this point soon